splash/proxy.py

```diff
@@ -104,6 +105,8 @@ def _getFilterParams(self, profile_name=None):
         Return (blacklist, whitelist, proxy_list) tuple
         loaded from profile ``profile_name``.
         """
+        if isinstance(profile_name, bytes):
```

Comment:
profile_name is always bytes whenever it is provided as the proxy keyword. I should move the decoding into render_options.py#125